src/Core/System/SystemConfig/Schema/config.xsd

```diff
@@ -12,13 +12,13 @@
         </xs:unique>
     </xs:element>
     <xs:complexType name="card">
-        <xs:sequence>
+        <xs:choice minOccurs="0" maxOccurs="unbounded">
             <xs:element name="title" type="translatableString" maxOccurs="unbounded"/>
```

Comment:
maybe title and name should still be at the beginning? 🤔 what do you think?

Reply:
If you specify title and name they still show at the top, most likely because of their static implementation in the template (I did not look into it in detail).

For further reading, here's a good explanation, but I think it is too verbose for the purpose.
https://stackoverflow.com/a/12012599/1705621